File: mathematical_operations/js/math_operations.js
//math_operations.js --- using node.js readline-sync module
//(must install the module to run --- package.json included) 
const rlSync = require('readline-sync');

const a = parseInt(rlSync.question("First integer: "));
const b = parseInt(rlSync.question("Second integer: "));

const sum = a + b;
const product = a * b;
const difference = a - b;
const quotient = a / b;
const remainder = a % b;


console.log("The sum %d + %d = %d", a, b, sum);
console.log("The difference %d - %d = %d", a, b, difference);
console.log("The product %d * %d = %d", a, b, product);
console.log("The quotient %d / %d = %d", a, b, quotient);
console.log("The remainder of the division %d / %d = %d", a, b, remainder);

File: mathematical_operations/cpp/math_operations.cpp
#include<iostream>

using namespace std;

int main()
{
  int a,b;
  cout<<"Please enter first number  : ";
  cin>>a;
  cout<<"Please enter second number : ";
  cin>>b;

  int sum = a + b;
  int product = a * b;
  int difference = a - b;
  int quotient = a / b;
  int remainder = a % b;

  cout<<"The sum of the two number is         : "<<sum<<endl;
  cout<<"The product of the two number is     : "<<product<<endl;
  cout<<"The difference between two number is : "<<difference<<endl;
  cout<<"The quotient of the two number is    : "<<quotient<<endl;
  cout<<"The remainder of the two number is   : "<<remainder<<endl;

  return 0;
}
